README.md

```diff
@@ -451,3 +451,13 @@ module('Integration | Component | foo', function (hooks) {
 v2.0 of the addon is built for Ember Octane (>= v3.17) and beyond. It contains breaking changes from the previous releases. If you would like to upgrade from v1.0 or earlier, please following the instructions in [UPGRADING_TO_v2.x.md](UPGRADING_TO_v2.x.md) file.
 
 <p align="center"><sub>Made with :heart: by The Ember Launch Darkly Team</sub></p>
+
+## Upgrading to v3.0
+
+Below you can find a list of deprecations, most of them should already be working correctly with modern ember apps:
+- Dropped node support below v16
+- Dropped ember support below LTS v3.28 and 4.4
+
+## Deprecations
```

Comment:
```suggestion
### Deprecations
```

I guess this is part of the Upgrading to v3.0 section

Reply:
Fixed!